core/node/via_btc_sender/src/btc_inscription_aggregator.rs

```diff
@@ -80,42 +80,42 @@ impl ViaBtcInscriptionAggregator {
             .await?
         {
             let mut transaction = storage.start_transaction().await?;
-
-            let inscription_message =
-                InscriptionMessage::to_bytes(&op.get_inscription_message().clone());
-
-            // Estimate the tx fee to execute the inscription request.
-            let inscribe_info = self
-                .inscriber
-                .prepare_inscribe(
-                    &op.get_inscription_message().clone(),
-                    InscriptionConfig::default(),
-                )
-                .await
-                .context("Get fee prepare inscriber")?;
-
-            let prediction_fee = inscribe_info.reveal_tx_output_info._reveal_fee
-                + inscribe_info.commit_tx_output_info._commit_tx_fee;
-
-            let inscription_request = transaction
-                .btc_sender_dal()
-                .via_save_btc_inscriptions_request(
-                    op.get_action_type(),
-                    inscription_message,
-                    prediction_fee.to_sat(),
-                )
-                .await
-                .context("via save btc inscriptions request")?;
-
-            transaction
-                .via_blocks_dal()
-                .set_inscription_request_id(
-                    op.get_l1_batch_metadata().header.number,
-                    inscription_request.id,
-                    op.get_action_type(),
-                )
-                .await
-                .context("set inscription request id")?;
+            for (index, message) in op.get_inscription_messages().clone().iter().enumerate() {
+                let inscription_message = InscriptionMessage::to_bytes(message);
+
+                // Estimate the tx fee to execute the inscription request.
+                let inscribe_info = self
+                    .inscriber
+                    .prepare_inscribe(message, InscriptionConfig::default())
```

Comment:
The InscriptionConfig is utilized for configuring BTC transactions with various settings. Currently, it allows us to set a gas multiplier for retry mechanisms. Since this is not part of the protocol configuration, I believe it's appropriate to keep it within the via_btc_client. wdyt?